core/src/test/resources/yaml/conf/deployment.yaml

```diff
@@ -0,0 +1,44 @@
+################################################################################
+#   Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+################################################################################
+  # Carbon Configuration Parameters
+wso2.carbon:
+  id: carbon-kernel
+  name: WSO2 Carbon Kernel
+  version: ${carbon.version}
```

Comment:
This is for the unit test. I change those values in deployment.yaml which is packed in feature. commit revision 05850e1c5482fffbe4efb4c802650ea7ec54cac9